Assets/BossRoom/Scripts/Editor/BakingMenu.cs

```diff
@@ -0,0 +1,148 @@
+using UnityEngine;
+using UnityEditor;
+using UnityEngine.Rendering;
+
+public class BakingMenu : MonoBehaviour
```

Comment:
Let's add a summary here of the what/why, since I originally didn't know that these operations only apply to currently loaded scene(s).